sos/report/plugins/gluster.py

```diff
@@ -55,6 +55,8 @@ def postproc(self):
 
     def setup(self):
         self.add_forbidden_path("/var/lib/glusterd/geo-replication/secret.pem")
+        self.add_forbidden_path("/var/lib/glusterd/glusterfind/"
+                                "glusterfind_*_secret.pem")
```

Comment:
Not really sure about this way of splitting the line, to be honest. Any way to improve it?

Reply:
```python
        self.add_forbidden_path(
            "/var/lib/glusterd/glusterfind/glusterfind_*_secret.pem"
        )
```